src/main/java/com/javadeobfuscator/deobfuscator/transformers/special/ParamorphismTransformer.java

```diff
@@ -359,6 +346,37 @@ else if(replacements.get(caller) instanceof FieldInsnNode)
 				}
 			}
 		}
+		classNodes().stream().forEach(cn -> {
+			cn.methods.stream().forEach(mn -> {
```

Comment:
Forgot to mention:  No need for an explicit `.stream()` here.

Reply:
emmm..... It doesn't matter. I think.